jobs/jobs.yml

```diff
@@ -83,10 +85,14 @@
         py: "3.11"
       - plone-version: "6.1"
         py: "3.8"
+      - plone-version: "6.0"
+        py: "3.13"
       - plone-version: "6.0"
         py: "3.12"
       - plone-version: "6.0"
         py: "3.10"
+      - plone-version: "5.2"
+        py: "3.13"
```

Comment:
Plone 5.2 is out of maintenance. We won't spend time supporting new versions of Python for it.

Reply:
Duh, this is the exclusions list. Never mind!